.github/workflows/build-docker.yml

```diff
@@ -11,7 +11,8 @@ on:
   pull_request:
     paths:
       - ".github/workflows/build-docker.yml"
-      - "docker/centrifuge-chain/Dockerfile"
+      - "docker/"
```

Comment:
This includes adjustments to the `docker-compose` files which we have to touch for every Substrate upgrade. I don't think this should trigger a docker build. Why is this change necessary?